src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -74,10 +102,10 @@ public void mecanumDrive(double joystickX, double joystickY, double rotation) {
 			backRightPower *= ControlConstants.MAX_ROBOT_SPEED / Math.abs(backRightPower);
 
 		// Power the motors
-		frontLeft.set(frontLeftPower);
-		frontRight.set(frontRightPower);
-		backLeft.set(backLeftPower);
-		backRight.set(backRightPower);
+		frontLeftController.set(frontLeftPower);
+		frontRightController.set(frontRightPower);
+		backLeftController.set(backLeftPower);
+		backRightController.set(backRightPower);
 	}
 
```

Comment:
add a method here to return encoder position and velocity - this will allow other subsystems and commands to access this information

This can look like
`public double getFrontLeftPosition()`, `public double getEncoderPosition(int index)`, where index is between 0 to 3 to correspond to one of the motors, etc